torch/utils/checkpoint.py

```diff
@@ -39,59 +39,60 @@ def backward(ctx, *args):
         return (None,) + tuple(inp.grad for inp in detached_inputs)
 
 
-def checkpoint(run_function, *args):
+def checkpoint(function, *args):
     r"""Checkpoint a model or part of the model
 
     Checkpoint works by trading compute for memory. It can be applied on any
     part of the model. In the forward pass, the model activations are not
-    stored. The forward pass save the inputs tuple and the run_function
-    parameter. In the backwards pass, the saved inputs and run_function is
-    retreived, and the forward pass is done on the model again (non-volatile
+    stored. The forward pass save the inputs tuple and the :attr:`function`
+    parameter. In the backwards pass, the saved inputs and :attr:`function` is
+    retreived, and the forward pass is done on the model again (`non`-volatile
```

Comment:
Can you please remove this reference to volatile? It might be confusing for new users, since we no longer explain this concept in the docs